### PR TITLE
added grid cell tooltip package

### PR DESCRIPTION
**Pull request by ritters**
@periklis @mmoll @elger @xalopp @andythedandy @calbrecht 
please review - sänk you and servus

### CODE REVIEW

File: packages/mf-grid-celltooltip/examples/Readme.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+- [Simple Grid](simple-grid/index.html) is a simple application, that uses the plugin feature.
+  You can also have a look at this example by using the PHP built-in web server:
+
+In your project root directory (ExtJsPackages)
+
+```sh
+    $ php -S localhost:65432
+```
+
+And open the [Simple Grid Example](http://http://localhost:65432/packages/mf-grid-celltooltip/examples/simple-grid/index.html).
```
Comment (mmoll) [inline packages/mf-grid-celltooltip/examples/Readme.md]: http://http://

**Comment by mmoll** [comment]
I can't test this :rotating_light: 

```
[ERR] loading page /home/mmoll/mystuff/gitrepos/ExtJsPackages/packages/mf-grid-celltooltip/examples/simple-grid/sass/example/theme.html
Failed to load page

[ERR] 
[ERR] BUILD FAILED
[ERR] com.sencha.exceptions.ExProcess: phantomjs process exited with code 100
```

**Comment by ritters** [comment]
this is caused by ignoring sass/example files.
@xalopp fix your .gitignore

**Comment by mmoll** [comment]
Maybe add a longer line into the example data set to get a glimpse of the usefullness of such a feature. :+1: otherwise.

**Comment by mmoll** [comment]
:dizzy:

**Comment by mmoll** [comment]
BTW, this needs to get squashed before merging.

**Comment by xalopp** [comment]
@mmoll @ritters looks fine to me.

**Comment by mmoll** [comment]
merged as f17f1d3, sänks @ritters!
...änd Servus!

**Pull request closed without merging**